### PR TITLE
add portugal states

### LINKED ISSUE #29206: Portugal States
https://github.com/woocommerce/woocommerce/blob/35d4ee4073a180d6fa07fa63a4fce37c394cfa1c/i18n/states.php#L1299

	'PT' => array( // Portugal states.
		'PT-01'   => __( 'Aveiro', 'woocommerce' ),
		'PT-02'   => __( 'Beja', 'woocommerce' ),
		'PT-03'   => __( 'Braga', 'woocommerce' ),
		'PT-04'   => __( 'Bragança', 'woocommerce' ),
		'PT-05'   => __( 'Castelo Branco', 'woocommerce' ),
		'PT-06'   => __( 'Coimbra', 'woocommerce' ),
		'PT-07'   => __( 'Évora', 'woocommerce' ),
		'PT-08'   => __( 'Faro', 'woocommerce' ),
		'PT-09'   => __( 'Guarda', 'woocommerce' ),
		'PT-10'  => __( 'Leiria', 'woocommerce' ),
		'PT-11'  => __( 'Lisboa', 'woocommerce' ),
		'PT-12'  => __( 'Portalegre', 'woocommerce' ),
		'PT-13'  => __( 'Porto', 'woocommerce' ),
		'PT-14'  => __( 'Santarém', 'woocommerce' ),
		'PT-15'  => __( 'Setúbal', 'woocommerce' ),
		'PT-16'  => __( 'Viana do Castelo', 'woocommerce' ),
		'PT-17'  => __( 'Vila Real', 'woocommerce' ),
		'PT-18'  => __( 'Viseu', 'woocommerce' ),
		'PT-20'  => __( 'Açores', 'woocommerce' ),
		'PT-30'  => __( 'Madeira', 'woocommerce' ),
	),

Reference: https://en.wikipedia.org/wiki/ISO_3166-2:PT

### PR DESCRIPTION
**Pull request by masteradhoc**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Add States based on Issue #29206
Closes #29206 .

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [-] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Localization - Add Portugal states.

### CODE REVIEW

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Thanks for your help.

But there's some issues with this change since is using ISO and based in the wikipedia instead of the CLDR as recommended in this file header.

Please replace all names to match with those: https://github.com/unicode-org/cldr/blob/release-38-1/common/subdivisions/en.xml#L4139-L4159

**Comment by masteradhoc** [comment]
@claudiosanches thanks for your comment. to help get this merged quickly i've made the adjustments and also fixed the codesnif - hopefully ;)

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Sorry, I meant only the names of the states, the previous codes as already fine, please replace `pt` to `PT-`.

File: i18n/states.php [hunk 1]
```diff
@@ -1296,7 +1296,28 @@
 	),
 	'PL' => array(),
 	'PR' => array(),
-	'PT' => array(),
+	'PT' => array( // Portugal states.
```
Comment (claudiosanches) [inline i18n/states.php:1299]:
```suggestion
	'PT' => array( // Portugal states. Ref: https://github.com/unicode-org/cldr/blob/release-38-1/common/subdivisions/en.xml#L4139-L4159
```

**Review by claudiosanches — APPROVED** [review]
Thank you very much!

**Pull request merged**